R/plotDataSetList.R

```diff
@@ -1366,10 +1388,10 @@ Plot.RT.Multi_Func.DataSetList <- function(dsList, scale.xlog = F,
   } else if (n_fcts <= 20) {
     n_rows <- ceiling(n_fcts / 3.)
     n_cols <- 3
-  } else if (n_fcts <= 30) {
+  } else {
```

Comment:
Shall we make a rule to calculate the number of rows and columns automatically?

Reply:
Sure, I will add this as a todo